pkg/auth/zcapld/service.go

```diff
@@ -48,13 +44,10 @@ func New(keyManager kms.KeyManager, crypto cryptoapi.Crypto, storeProv ariesstor
 		return nil, fmt.Errorf("failed to open store %s: %w", storeName, err)
 	}
 
-	ldDocumentLoader, err := createJSONLDDocumentLoader()
-	if err != nil {
-		return nil, fmt.Errorf("failed create json ld document loader: %w", err)
-	}
+	// TODO re-enable cached jsonld document loader: https://github.com/trustbloc/edv/issues/173.
```

Comment:
this performance killer

Reply:
Hi @fqutishat 

Turns out the framework is adding a cache anyway (see #173 ).

Reply:
[This is where it's added](https://github.com/hyperledger/aries-framework-go/blob/bf8af862aa8358f2a91cb0d2856bd366919f795b/pkg/doc/signature/jsonld/processor.go#L264-L272).

The cache map is passed in [here](https://github.com/hyperledger/aries-framework-go/blob/bf8af862aa8358f2a91cb0d2856bd366919f795b/pkg/doc/signature/proof/jws.go#L123-L122).

[These](https://github.com/hyperledger/aries-framework-go/blob/bf8af862aa8358f2a91cb0d2856bd366919f795b/pkg/doc/signature/proof/cache.go#L9-L13) are the cached JSON-LD contexts of the framework. Notice it has both context we need (https://w3id.org/security/v1, https://w3id.org/security/v2).

Reply:
@fqutishat please check now - the JSONLD documents are cached, but a new `CachingDocumentLoader` is used for every verification.